### ISSUE TITLE
Prebuilt binaries

### ISSUE THREAD

**Issue by sn-o-w** [RustPython/RustPython#5440]
Hi there!

Do you plan to create prebuilt (automated) binaries in the foreseeable future?

I'm running low on storage space on my computer, and it would be nice to be able to install RustPython easily without needing to build it ourselves. 😊

Thank you in advance.

**Comment by youknowone** [comment]
This is a good suggestion. Could anyone please send a patch of github actions release?

**Comment by theshubhamp** [comment]
@youknowone / @sn-o-w - saw this issue open and sent a PR: https://github.com/RustPython/RustPython/pull/5456

Could you folks take a look ? Cutting a new release on every merge to main for now - do we want to do something different ?

**Comment by theshubhamp** [comment]
Ah, so at the moment release is only creating interpreter binaries without the full stdlib wired in. 

You should be able to download the binary and run it as-is. But packaging in the stdlib needs more work atop this.

**Comment by arihant2math** [comment]
I don't think it is possible to wire the stdlib in until there is an actual installer that installs the executable in a folder with the stdlib and then setting the right variables to point at the stdlib.

**Comment by arihant2math** [comment]
Fixed by #5549.

**Comment by jayvdb** [comment]
#5549 was merged but there are no binaries at https://github.com/RustPython/RustPython/releases/tag/0.5.0

I suspect the problem is there are zero (0) releases at https://github.com/RustPython/RustPython/releases

**Comment by jayvdb** [comment]
cargo-quickinstall has prebuilt binaries at https://github.com/cargo-bins/cargo-quickinstall/releases/tag/rustpython-0.5.0

**Comment by jayvdb** [comment]
https://www.npmjs.com/package/rustpython_wasm is 0.1.0-pre-alpha.2 , published 7 years ago in 2019

**Comment by jayvdb** [comment]
I've published rustpython_wasm and a few platforms of rustpython (with feature freeze-stdlib) at https://github.com/edge-toolkit/core/releases/tag/rp-v1
